code/modules/mining/ore_redemption_machine/survey_vendor.dm

```diff
@@ -42,7 +42,8 @@
 		new /datum/data/mining_equipment("Defense Equipment - Razor Drone Deployer",/obj/item/weapon/grenade/spawnergrenade/manhacks/station,	50),
 		new /datum/data/mining_equipment("Defense Equipment - Sentry Drone Deployer",/obj/item/weapon/grenade/spawnergrenade/ward,			100),
 		new /datum/data/mining_equipment("Defense Equipment - Steel Machete",	/obj/item/weapon/material/knife/machete,					50),
-		new /datum/data/mining_equipment("Survival Equipment - Insulated Poncho",	/obj/random/thermalponcho,								75)
+		new /datum/data/mining_equipment("Survival Equipment - Insulated Poncho",	/obj/random/thermalponcho,								75),
+		new /datum/data/mining_equipment("Survey Tools - Compact Catalouger",				/obj/item/device/cataloguer/compact,				50),
```

Comment:
prize_list is not actually used at all. I am not sure why it exists. Also Compact cataloguers should be in a supplypack, I would recommend making a pilot supply pack alongside the explorer one, since the pilot locker has a few unique things as awell.

Reply:
I don't know what prize_list is, I've just been copy and pasting code and it just works